pages/deposit-dates.css

```diff
@@ -0,0 +1,49 @@
+/* TODO: Custom media from layout.css doesn't work */
+@custom-media --screen-until-desktop (max-width: 727px);
+
+.table thead > :first-child {
+  border-top-width: 2px;
+}
+
+.row {
```

Comment:
I expected to have a single flex container that is actually main. Do you mind if I play with styles a little bit more?

Reply:
Go ahead. You are a CSS Pro. 🙂

Reply:
Check #41 🙂